Makefile

```diff
@@ -3,6 +3,9 @@ NODE_MODULES = node_modules
 PROJECT = js-libjuju
 SYSDEPS = build-essential python3-virtualenv tox
 
+ADMIN_FACADE = api/facades/admin-v3.js
```

Comment:
I'm assuming this is used as a canary? Could you add a comment to that effect for others.

Reply:
Done.